### PR TITLE
add bcc 0.12.0 support

### PR DESCRIPTION
**Pull request by brayniac**
Adds support for bcc 0.12.0 via feature flag and simplifies ci
build script

### CODE REVIEW

File: build/ci.sh [hunk 1]
```diff
@@ -0,0 +1,78 @@
+#!/bin/bash -ev
+
+## Install toolchain
+if [[ "${TRAVIS_DIST}" == "xenial" ]]; then
+    if [[ "${LLVM_VERSION}" == "8" ]]; then
```
Comment (WesleyAC) [inline build/ci.sh]: Could you condense these into one, and just use the ${LLVM_VERSION} variable in the commands? seems like that shouldn't be too hard.

File: build/ci.sh [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    mkdir -p _build
+    cd _build
+    cmake .. -DCMAKE_INSTALL_PREFIX=/usr
+    make
+    sudo make install
+    cd ../..
+fi
+
+## Build and test
+if [ -z "${FEATURES}" ]; then
```
Comment (WesleyAC) [inline build/ci.sh]: Same comment here - maybe this is a little more annoying to do right, but it seems like these could be condensed. (Maybe rewrite the FEATURES var to include the `--features`, then just use that?)

File: build/ci.sh [hunk 3]
```diff
@@ -0,0 +1,78 @@
+#!/bin/bash -ev
```
Comment (WesleyAC) [inline build/ci.sh]: run shellcheck on this? nothing serious in the output, but might as well :)

File: build/ci.sh (hunk 2 above)
Comment (brayniac) [reply build/ci.sh]: Fixed

File: build/ci.sh (hunk 3 above)
Comment (brayniac) [reply build/ci.sh]: Added in the travis.yaml

File: build/ci.sh [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+## Install toolchain
+if [[ "${TRAVIS_DIST}" == "xenial" ]]; then
+    wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key|sudo apt-key add -
+    echo "deb http://apt.llvm.org/xenial/ llvm-toolchain-xenial-${LLVM_VERSION} main" | sudo tee -a /etc/apt/sources.list
+    sudo add-apt-repository --yes ppa:ubuntu-toolchain-r/ppa
+    sudo apt-get update
+    sudo apt-get install linux-headers-"$(uname -r)"
+    sudo apt-get --yes install bison build-essential cmake flex git libclang-common-"${LLVM_VERSION}"-dev libelf-dev libllvm"${LLVM_VERSION}" libz-dev lldb-"${LLVM_VERSION}" llvm-"${LLVM_VERSION}" llvm-"${LLVM_VERSION}"-dev llvm-"${LLVM_VERSION}"-runtime
```
Comment (WesleyAC) [inline build/ci.sh]: Sort of nitpicky, but could you have only this command be in the if statement? You can use ${TRAVIS_DIST} in the line adding it to sources.list.

**Pull request merged**